Describe socket recv buffer bug (#2)
jellyfish-dev/book: src/bugs/README.md

> # The Great Chapter of Bugs

Reviewer: How about incorporating a :hankey: emoji somewhere around here?

Author: This might be too much :smiley:

Author: I mean, the transition from `days or weeks` to  `weeks or months` should be enough

> During the development of Jellyfish, we encountered a wide variety of bugs which solving sometimes took us

Reviewer: ```suggestion
During the development of Jellyfish, we encountered a wide variety of bugs that took weeks or months to solve.
```

days or weeks.
This chapter collects the most interesting ones, explaining where was the problem and how we solved it.

> We hope that you will find here things that we couldn't find anywhere.

Reviewer: ```suggestion
This chapter collects the most interesting ones, explaining where the problem was and how we solved it.
We hope that you will find here things that we couldn't find anywhere.

**WARNING!!!**: This chapter contains extremely disturbing content. If you have weak nerves, don't read.
```
